orchestrator/main_loops.go

```diff
@@ -504,7 +504,7 @@ func getEthBlockDelay(chainID uint64) uint64 {
 	// Mainline Ethereum, Ethereum classic, or the Ropsten, Kotti, Mordor testnets
 	// all POW Chains
 	case 1, 3, 6, 7:
-		return 13
+		return 96
```

Comment:
Could keep 13 for known testnets and use `96` for chain ID `1` and the `default` case.

Reply:
I talked too soon, the E2E test uses chain id `15`. So the E2E shouldn't have any issues. Let me separate it in its own case anyway